### ISSUE TITLE
Feature request: show rendered HTML beside code examples in survey

### ISSUE THREAD

**Issue by diamondavocado** [Devographics/Monorepo#319]
In many survey questions there are blocks of example HTML/JS code.

These examples should be shown rendered on the page.

It is not always easy to recognise the features being mentioned when you can only see the source code.

**Comment by SachaG** [comment]
I agree this would be cool, but I'm worried it would only work for a few features, since a lot of them don't really have a visual representation. It would also risk making the survey itself very cluttered visually.
